src/SwedbankPay.Episerver.Checkout.Common/Constants.cs

```diff
@@ -6,14 +6,12 @@ public static class Constants
 
         public const string SwedbankPayCheckoutSystemKeyword = SwedbankPaySystemKeyword + "Checkout";
 
-        // Cart property fields
-        public const string SwedbankPayCheckoutOrderIdCartField = "SwedbankPayCheckoutOrderIdCart";
-
         // Payment method property fields
         public const string SwedbankPaySerializedMarketOptions = "SwedbankPaySerializedMarketOptions";
 
         // Purchase order meta fields
         public const string SwedbankPayOrderIdField = "SwedbankPayOrderId";
         public const string SwedbankPayPaymentIdField = "SwedbankPayPaymentId";
+        public const string SwedbankPayPayeeReference = "SwedbankPayPayeeReference";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-episerver-checkout/pullRequest?prid=5328029)